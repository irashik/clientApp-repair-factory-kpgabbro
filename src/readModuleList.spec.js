import React from 'react';

import ReadModuleList  from './readModuleList';

       


describe("<ReadModuleList/>", () => {
    test('should render correctly empty list', () => {

        const component = shallow(<ReadModuleList />)
        expect(component).toMatchSnapshot();
    });



    test('should render correctly full list', () => {


        // добавь данные ...

        const component = shallow(<ReadModuleList />)

        expect(component).toMatchSnapshot();


    });




});



