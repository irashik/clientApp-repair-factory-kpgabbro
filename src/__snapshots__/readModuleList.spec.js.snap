// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReadModuleList/> should render correctly empty list 1`] = `
<div>
  <h2>
    Нет записей
  </h2>
</div>
`;

exports[`<ReadModuleList/> should render correctly full list 1`] = `
<div>
  <h2>
    Нет записей
  </h2>
</div>
`;
